pkgs/data/fonts/apple-fonts/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib, stdenv, fetchurl, unzip, p7zip }:
+
+stdenv.mkDerivation rec {
+  pname = "apple-fonts";
+  version = "1.0.3";
```

Comment:
Where does this version come from?

Reply:
it's coming from the AUR package they have maintained, wasn't sure how this would be, so i just kept the same value